templates/mail/self-serve/connection-info.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'mail/self-serve/_base.html' %}
+{% block page_heading %}
+  <h2>Self Serve - Connection Information</h2>
+{% endblock %}
+{% block body %}
+  <div class="connection-info-container">
+    <div class="receiving-email">
+      <h3>Receiving Email</h3>
+      <div>
+        <h4 id="imap">IMAP</h4>
+        <p><b>Server Name:</b> {{ IMAP.HOST }}</p>
+        <p><b>Server Port:</b> {{ IMAP.PORT }}</p>
+        <p><b>Security:</b> {% if IMAP.TLS %}SSL/TLS{% else %}None{% endif %}</p>
+        <p><b>Username:</b> {{ mail_username }}</p>
+        <p><b>Password:</b> Your App Password</p>
+      </div>
+      <div>
+        <h4 id="jmap">JMAP</h4>
+        <p><b>Server Name:</b> {{ JMAP.HOST }}</p>
+        <p><b>Server Port:</b> {{ JMAP.PORT }}</p>
+        <p><b>Security:</b> {% if JMAP.TLS %}SSL/TLS{% else %}None{% endif %}</p>
+        <p><b>Username:</b> {{ mail_username }}</p>
+        <p><b>Password:</b> Your App Password</p>
+      </div>
+    </div>
+    <div class="sending-email">
+      <h3>Sending Email</h3>
+      <div>
+        <h4 id="smtp">SMTP</h4>
+        <p><b>Server Name:</b> {{ SMTP.HOST }}</p>
+        <p><b>Server Port:</b> {{ SMTP.PORT }}</p>
+        <p><b>Security:</b> {% if SMTP.TLS %}SSL/TLS{% else %}None{% endif %}</p>
+        <p><b>Username:</b> {{ mail_username }}</p>
+        <p><b>Password:</b> Your App Password</p>
+      </div>
+    </div>
+  </div>
+{% endblock %}
```

Comment:
Thinking about splitting username/password from each block. I think this is all the information someone may need though.